lib/datadog/core/environment/variable_helpers.rb

```diff
@@ -4,6 +4,8 @@ module Core
     module Environment
       # Defines helper methods for environment
       module VariableHelpers
+        extend self
```

Comment:
Note: This enables use of the module helpers without needing to include the module.